src/viam/sdk/services/motion/motion.cpp

```diff
@@ -135,8 +135,8 @@ bool operator==(const obstacle_detector& lhs, const obstacle_detector& rhs) {
 
 std::ostream& operator<<(std::ostream& os, const obstacle_detector& v) {
     os << "{ ";
-    os << "\tvision_service: " << v.vision_service << std::endl;
-    os << "\tcamera: " << v.camera << std::endl;
+    os << "\tvision_service: " << v.vision_service << "\n";
```

Comment:
Why is `std::endl` disallowed?

Reply:
Ah so strange; I'm surprised the compiler can't just optimize there and _not_ flush with `std::endl` when using one of the standard library streams that automatically flushes on every write. Thanks for the context!